server/routes/bookedtoday/arrivals/checkAnswersController.ts

```diff
@@ -61,7 +61,7 @@ export default class CheckAnswersController {
       req.flash('arrivalResponse', {
         firstName: arrival.firstName,
         lastName: arrival.lastName,
-        prisonNumber: arrival.prisonNumber,
+        prisonNumber: arrivalResponse.prisonNumber,
```

Comment:
changed to obtain prisonNumber from arrivalResponse as this no match journey has an undefined prisonNumber in arrival object which was causing a redirect to the choose prisoner page in the confirmation controller as prisonNumber was undefined in flash.

Reply:
yep that makes sense